### PR TITLE
Newui

### PR DESCRIPTION
**Pull request by ThomasThelen**
This adds a checkbox on the registration modal dialog that appears when the server is deployed in a development environment.

A flag was added to `environment.js` which is then checked in the handlebars template to determine whether the checkbox should appear.

### CODE REVIEW

File: app/components/ui/files/registration-modal/component.js [hunk 1]
```diff
@@ -108,9 +110,15 @@ export default Ember.Component.extend({
     },
 
     actions: {
+
+        updateDev(value) {
+            // Called if the `use dev` checkbox is clicked
+            this.set('useDev', value)
```
Comment (cyberdante) [inline app/components/ui/files/registration-modal/component.js]: This is really minor, @ThomasThelen, but I favor ending lines with a semicolon as a rule that I even try enforcing with my editor.

File: app/components/ui/files/registration-modal/component.js [hunk 2]
```diff
@@ -174,10 +188,17 @@ export default Ember.Component.extend({
             Ember.$('#results').removeClass('hidden');
```
Comment (cyberdante) [inline app/components/ui/files/registration-modal/component.js]: I also avoid using jQuery as much as possible. I favor having variables and rendering the handlebars/html-bars sections using these variables, in this case, something like:

```
    {{#if resultsVisible}}
        <div id="results"> 
            ...
        </div>
    {{/if}}
```

Comment (ian-taylor) [reply app/components/ui/files/registration-modal/component.js]: I agree - there are a number of additional handlebars rules we have included in WholeTale using the ember truth helpers too that allow you to do quite complex logic in the templates directly: https://github.com/jmurphyau/ember-truth-helpers

Comment (cyberdante) [reply app/components/ui/files/registration-modal/component.js]: @ThomasThelen, where are we on this? Can I merge and close this PR already?

Comment (ThomasThelen) [reply app/components/ui/files/registration-modal/component.js]: Ready to go!

**Pull request merged**